plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt

```diff
@@ -54,20 +50,7 @@ open class KtlintBasePlugin : Plugin<Project> {
         )
     }
 
-    /**
-     * @deprecated Now that we declare gradle API metadata, this code should not be needed.
-     * Ee need to check which version of gradle introduced gradle API metadata checking
-     */
-    @Deprecated("Now that we declare gradle API metadata, this code should not be needed")
-    private fun Project.checkMinimalSupportedGradleVersion() {
```

Comment:
From what version is this metadata standard supported from?
Are we sure we want to yank this out? It seems like a reasonable sanity check if someone does something silly and applies the plugin to an ancient version of Gradle

Reply:
Actually the reason I chose to yank this now is that any version of gradle lower than 7.6 + kotlin 2.x causes the build to fail in a different built-in way (with a nice message). So i stopped testing against 7.4. But it still works as long as you dont use kotlin 2 x, so I left the api compatibility declaration at 7.4.
But to test this method now, we would need a specific test that uses gradle 7.4 and kotlin 1.x 
I can add that and reinstate this function.

Reply:
Please do, if you have time and it's not too much of a headache